docs/syncer.md

```diff
@@ -11,77 +11,79 @@ users have to install a special component named [syncer](https://github.com/kcp-
 
 ## Instructions
 
-1. Create a workspace
+1. Create a workspace and immediately enter it:
 
 ```sh
-$ kubectl kcp workspace create my-workspace --use
-Workspace "my-workspace" created.
-Current workspace is "my-workspace".
+$ kubectl kcp workspace create my-workspace --enter
+Workspace "my-workspace" (type "Universal") created. Waiting for being ready.
+Current workspace is "root:default:my-workspace".
 ```
 
-Check that it has been created correctly and is ready:
+1. Enable the syncer for a new cluster
 
 ```sh
-$ kubectl kcp workspace list
-NAME           TYPE        PHASE   URL
-my-workspace   Universal   Ready   https://192.168.1.150:6443/clusters/default:my-workspace--1
+$ kubectl kcp workload sync <mycluster> --syncer-image <image name> > syncer.yaml
 ```
 
-Obtain the KCP server URL and the logical cluster name:
-
-- KCP_SERVER_URL=https://192.168.1.150:6443
-- KCP_LOGICAL_CLUSTER_NAME=default:my-workspace--1
-
-2. Create a service account for the syncer on current workspace:
+1. Create a kind cluster to back the workload cluster
 
 ```sh
-$ kubectl create serviceaccount syncer
-serviceaccount/syncer created
+$ kind create cluster
+Creating cluster "kind" ...
+<snip>
+Set kubectl context to "kind-kind"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kind
 ```
 
-3. Generate a kubeconfig from the service account:
+## For syncer development
 
-```sh
-$ kubectl get secret
-NAME                  TYPE                                  DATA   AGE
-default-token-nkpxj   kubernetes.io/service-account-token   3      29m
-syncer-token-2mtgf    kubernetes.io/service-account-token   3      29m
+Alternately, create a `kind` cluster with a local registry to simplify syncer development by executing the
+following script:
 
-```
+https://raw.githubusercontent.com/kubernetes-sigs/kind/main/site/static/examples/kind-with-registry.sh
```

Comment:
what is special about that script? Why is the first way not enough?

Reply:
The script deploys kind with a registry, which is ideal for building the image locally and testing it without having to push it to a regsitry.